lib/nitcorn/examples/src/htcpcp_server.nit

```diff
@@ -0,0 +1,67 @@
+import nitcorn
```

Comment:
Every Nit file has a license at the beginning of the file, could you add one ?
You can copy-paste a license file from any other example in the nitcorn examples, otherwise, tests will not pass